### ISSUE TITLE
Compile error: find_seq_in_lookup_table: seq_number not found (shared/cfe/edgcpfe/il.c)

### ISSUE THREAD

**Issue by ktwalker** [intel/yask#198]
Hello,

I am interested in using YASK, but I am having difficulties getting started.  I am experienced in MPI, OpenMP, makefiles, C++, and scripts.  

I have followed the instructions in one of your tutorial PDF’s for building the iso3dfd stencil.  However, after some time in compiling, I end up with this error:

make stencil=iso3dfd arch=bdw

mpiicc -g -std=c++11 -Wall -I/home/kwdh/YASK/yask-master/include -I/home/kwdh/YASK/yask-master/src/kernel/lib -I/home/kwdh/YASK/yask-master/src/common -I/home/kwdh/YASK/yask-master/build/kernel/yask_kernel.iso3dfd.bdw/gen -DDEF_ARGS='"-thread_divisor 2 -block_threads 1 -d 1024 -bx 296 -by 5 -bz 290 -ep 1   "' -xCORE-AVX2 -debug extended -restrict -ansi-alias -fno-alias -fimf-use-svml=true -fimf-precision=low -fast-transcendentals -no-prec-sqrt -no-prec-div -fp-model fast=2 -fno-protect-parens -ftz -fma -fimf-domain-exclusion=none -qopt-assume-safe-padding -no-diag-message-catalog -vec-threshold0 -O3 -fopenmp -DMAX_EXCH_DIST=1 -DUSE_INTRIN256 -DYK_CODE_FILE=/home/kwdh/YASK/yask-master/build/kernel/yask_kernel.iso3dfd.bdw/gen/yask_stencil_code.hpp -DUSE_NUMA -DUSE_NUMAIF_H -DPFD_L1=0 -DPFD_L2=0 -DALLOW_NEW_GRIDS=1 -DARCH_BDW -DUSE_MPI -DUSE_SVML  -x c++ -fPIC -c -o /home/kwdh/YASK/yask-master/build/kernel/yask_kernel.iso3dfd.bdw/obj/settings.o /home/kwdh/YASK/yask-master/src/kernel/lib/settings.cpp
icc: command line warning #10006: ignoring unknown option '-fimf-use-svml=true'
Internal error loop: assertion failed: find_seq_in_lookup_table: seq_number not found (shared/cfe/edgcpfe/il.c, line 4094)

Do you know what this is reference to?

Kind regards,
Kris

**Comment by ktwalker** [comment]
I submitted an email to Chuck Yount.  He suggested it was related to the version of icc being used.  I was using:

> mpiicc -v
mpiicc for the Intel(R) MPI Library 2017 Update 2 for Linux*
Copyright(C) 2003-2017, Intel Corporation.  All rights reserved.
icc version 17.0.2 (gcc version 7.3.0 compatibility)

I switched to version 2019 and the problem is now fixed.  

> mpiicc -v
mpiicc for the Intel(R) MPI Library 2019 Update 1 for Linux*
Copyright 2003-2018, Intel Corporation.
icc version 19.0.1.144 (gcc version 7.3.0 compatibility)

Many thanks Chuck!
Kris
